crates/lib/kajiya/src/world_renderer.rs

```diff
@@ -858,8 +850,10 @@ impl WorldRenderer {
             .instances
             .iter()
             .flat_map(|inst| {
-                let inst_position = inst.position;
-                let inst_rotation = inst.rotation;
+                let (_scale, rotation, translation) =
+                    inst.transformation.to_scale_rotation_translation();
+                let inst_position = translation;
+                let inst_rotation = roation;
```

Comment:
Still one more roation left!
```suggestion
                let inst_rotation = rotation;
```